Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -161,17 +161,20 @@ extension File {
         }.map { $0.0 }
     }
 
+    internal typealias MatchMapping = (NSTextCheckingResult) -> NSRange
```

Comment:
Because I needed to get the capture group (`$0.rangeAt(1)`), and not the full match. I'll comment on the relevant line.

I've created the typealias just to save space to avoid an ugly formatting.